src/frequenz/quantities/experimental/marshmallow.py

```diff
@@ -57,24 +71,34 @@ class _QuantityField(fields.Field):
     field_type: Type[Quantity] | None = None
     """The specific Quantity subclass."""
 
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the field."""
+        self.serialize_as_string_override = kwargs.pop("serialize_as_string", None)
+        super().__init__(*args, **kwargs)
+
     def _serialize(
-        self, value: Quantity, attr: str | None, obj: Any, **kwargs: Any
+        self, value: Any | None, attr: str | None, obj: Any, **kwargs: Any
```

Comment:
This is redundant (`Any | None == Any`):

```suggestion
        self, value: Any, attr: str | None, obj: Any, **kwargs: Any
```

Why can't it be `Quantity | None`?